en/developer_guide/block_override.rst

```diff
@@ -0,0 +1,105 @@
+Block Override
+===================
+
+Overloading the template of a block
+===================================
+
+Overloading the template of a block is achieved by creating a template in the following folder structure :
+
+.. code-block::
+
+    |_ app
+        |_ Resources
+            |_ {BUNDLE_NAME}
+                |_ views
+                    |_ {PATH_TEMPLATE}
+                        |_ {FILENAME}
+
+
+Example :
+---------
+
+You need to override the display template of block "Content List Block" in Backoffice to display datas
```

Comment:
override the template of the `Content List Block` block from the Back Office to display additional datas.